solr/config/configsets/all_collection_oag56_v1/managed-schema

```diff
@@ -465,6 +466,7 @@
   <field name="author_pids" type="strings" required="false" useDocValuesAsStored="true"/>
   <field name="author_types" type="strings" uninvertible="true" docValues="true" multiValued="true" indexed="true" stored="true"/>
   <field name="best_access_right" type="string" required="false"/>
+  <field name="bestaccessright" type="text_general"/>
```

Comment:
Have you added this field by yourself? Is it a necessary field? It seems not. If that is the case, delete it.

Reply:
It was already in the schema which I copied. I removed it.